projects/crud/src/lib/components/model-form/model-form.component.html

```diff
@@ -9,7 +9,7 @@
                 <div class="header_btn_container" *ngIf="viewConfig?.metadata?.formActions && mode==='edit'">
                     <span class="header_btns">
                         <ng-container>
-                            <button mat-raised-button id="capture-btn" *ngIf="captureIdButton?.isShown" class="header_btn"
```

Comment:
Is it ok to remove this ID?
Did you check that we don't need it in the automation?